src/_premake_init.lua

```diff
@@ -1348,6 +1344,13 @@
 	end)
 
 
+	-- 31 January 2017
+
+	api.deprecateValue("flags", "C++11", 'Use `language "C++11"` instead', function(value) end, function(value) end)
+	api.deprecateValue("flags", "C++14", 'Use `language "C++14"` instead', function(value) end, function(value) end)
+	api.deprecateValue("flags", "C90",   'Use `language "C90"` instead',   function(value) end, function(value) end)
+	api.deprecateValue("flags", "C99",   'Use `language "C99"` instead',   function(value) end, function(value) end)
+	api.deprecateValue("flags", "C11",   'Use `language "C11"` instead',   function(value) end, function(value) end)
```

Comment:
Add a `Default` value to `language`? I'm picturing a situation like this:
```lua
project "NiceProject"
  language "C++14"
  files { "**.h", "**.c" }

  filter { "files:RealCFiles/**.c" }
    language "Default"
```

Would the language value flow down regardless of extension? Assuming it would, the above might be desired functionality?